p2p/swarm.go

```diff
@@ -233,6 +234,53 @@ func (s *swarm) SendMessage(peerPubkey p2pcrypto.PublicKey, protocol string, pay
 	return s.sendMessageImpl(peerPubkey, protocol, service.DataBytes{Payload: payload})
 }
 
+func (s *swarm) RetrySend(peerPubKey p2pcrypto.PublicKey, protocol string, payload service.Data) error {
```

Comment:
even though retrying seems to get the effect we want, please try doing this avoiding copying the whole func. 
also checkout latest changes from `develop`

Reply:
@y0sher since the signed encoded message does not change, but the whole process of lookup in the dht and `GetConnection` to establish a connection will remain the same if we have to retry because the connection might be stale. Can this be the definition? `RetrySend(peerPubKey p2pcrypto.PublicKey, message []byte)` where `message` is the `final` signed encoded message